package.json

```diff
@@ -73,10 +73,10 @@
     "lint:fix": "eslint --fix --resolve-plugins-relative-to \"./node_modules/@ridedott/eslint-config\" \"**/*.ts\"",
     "semantic-release": "semantic-release",
     "spellcheck": "cspell \"**/*\"",
-    "test": "jest",
-    "test:ci": "jest --ci --collect-coverage --maxWorkers=12",
-    "test:coverage": "jest --collect-coverage",
-    "test:watch": "jest --watch --verbose false",
+    "test": "jest --runInBand",
+    "test:ci": "jest --runInBand --ci --collect-coverage",
+    "test:coverage": "jest --runInBand --collect-coverage",
+    "test:watch": "jest --runInBand --watch --verbose false",
```

Comment:
Whoa, we can't do that - it disables parallelization!

Reply:
fixed with: https://github.com/ridedott/eslint-plugin/pull/30/commits/7533d0d520d776bd443a3f218d797e582b40e1d3